pkgs/development/libraries/libmbim/default.nix

```diff
@@ -9,15 +9,16 @@
 , help2man
 , systemd
 , bash-completion
-, withIntrospection ? stdenv.hostPlatform == stdenv.buildPlatform
```

Comment:
we should do `withIntrospection ? stdenv.hostPlatform.emulatorAvailable buildPackages` for cross-os compilation (like linux -> *BSD

Reply:
https://github.com/NixOS/nixpkgs/pull/220499